lib/js/components/RichList/RichListItem/RichListItem.vue

```diff
@@ -610,6 +615,7 @@ export default defineComponent({
 				'-ds-verticalWithMedia': this.isVertical && this.hasMedia,
 				'-ds-vertical': this.isVertical,
 				'-ds-loading': this.state === RICH_LIST_ITEM_STATE.LOADING,
+				'-ds-drag': this.state === RICH_LIST_ITEM_STATE.DRAG,
```

Comment:
Should we somehow limit this state? e.x. when component is not dragable or is not interactive or maybe other rules?